documentation/commands/alias.md

```diff
@@ -1,15 +1,21 @@
-#### NAME
+<h1 textrun="command-heading">Alias command</h1>
 
-alias - add or remove the default git aliases
+<blockquote textrun="command-summary">
```

Comment:
Since it appears summary and description come from the command, should we try to make these generated files where we can supply templates when we want to add more information?

Reply:
I have thought about that, but have decided against it for the following reasons:
1. the Markdown pages contain a good amount of custom formatting that exceeds that the CLI pages contain. For example links, or formatting parts of text as code blocks
2. this stuff changes pretty rarely. Creating a lot of infrastructure to parse the CLI output and transform it into markdown feels over-engineered and causes maintainability issues down the road. 

A verifier as implemented in this PR is much simpler, more robust, more maintainable, and allows some degree of freedom in designing the Markdown pages.